openlibrary/templates/account/loans.html

```diff
@@ -55,6 +55,9 @@
 <div>
   $if len(loans) == 0:
     <div><em>$_("You've not checked out any books at this moment.")</em></div>
+    $:render_template("books/custom_carousel", books=readonline_carousel(), title=_('Classic Books'), url="/read", key="public_domain", test=test, compact_mode=True)
```

Comment:
```suggestion
    <div>Looking for a book to borrow?  Check out our staff picks, or search for a book on a specific category:</div>
     $:render_template("home/custom_ia_carousel", title=_('Books We Love'), key="staff_picks", query='languageSorter:("English")', subject="openlibrary_staff_picks", sorts=["lending___last_browse desc"], limit=18, test=test, compact_mode=True)
```
I think that this is missing a call to action.  Let's add that before any of the carousels.  You may want to add a bit of space between the "You've not checked out any books at this moment." message and the call to action (unless you think that it looks good).

Although we can't easily test the "Books We Love" carousel locally, we can test this by deploying to https://testing.openlibrary.org.  Let's try replacing the "Classic Books" carousel with the "Books We Love" carousel.  Once that change has been made, I'll deploy this branch to testing.